packages/components/psammead-podcast-promo/src/components/title.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { node } from 'prop-types';
+
+import {
+  GEL_SPACING_HLF,
+  GEL_SPACING_DBL,
+} from '@bbc/gel-foundations/spacings';
+import { getGreatPrimer } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { C_EBON } from '@bbc/psammead-styles/colours';
+
+import PodcastIcon from '../icons/podcast';
```

Comment:
should we have icons in psammead assets?

Reply:
👍 https://github.com/bbc/psammead/pull/4009